speech/snippets/multichannel.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Package snippets contains speech examples.
+package snippets
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	// [START imports]
```

Comment:
Is this region tag going to be used? If so, the region tag should start with "speech" (required by the samples tracker).

Reply:
reordered the imports